src/main/resources/crafter/studio/upgrade/pipelines.yaml

```diff
@@ -463,6 +463,14 @@ pipelines:
         operations:
           - type: dbScriptUpgrader
             filename: upgrade-4.0.1.13-to-4.0.1.14.sql
+      - currentVersion: 4.0.1.14
```

Comment:
Please pull the latest and update the version number. There is already a v4.0.1.15

Reply:
Updated to new version `4.0.1.16`. Thanks.